.evergreen/auth_aws/activate-authawsvenv.sh

```diff
@@ -32,10 +32,7 @@ activate_authawsvenv() {
   else
     # shellcheck source=.evergreen/find-python3.sh
     . ../find-python3.sh || return
-
-    echo "Finding Python3 binary..."
-    PYTHON="$(find_python3 2>/dev/null)" || return
-    echo "Finding Python3 binary... done."
+    PYTHON=$(ensure_python3) || return
```

Comment:
Unsure if [this test failure](https://parsley.mongodb.com/evergreen/mongo_c_driver_aws_ubuntu2204_test_aws_openssl_ecs_latest_patch_eae52c0f3d30be7bbb0029eea878ede649253e02_66ce15e4b8ad300007e7d2a7_24_08_27_18_07_34/0/task?bookmarks=0,1182&shareLine=1130) is related to changes in this PR.

Reply:
It looks like removing `rm -rf mongodb` will fix this one too: `/tmp/test.sh: line 15: /root/mongosh: No such file or directory`.